src/Layout/redist/redist.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="NuGet.ProjectModel" Version="$(NuGetProjectModelPackageVersion)" />
     <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="$(MicrosoftNetCompilersToolsetPackageVersion)" ExcludeAssets="All" />
     <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="$(MicrosoftCodeAnalysisNetAnalyzersVersion)" ExcludeAssets="All" GeneratePathProperty="true"/>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeStyle" Version="$(MicrosoftCodeAnalysisCSharpCodeStylePackageVersion)" ExcludeAssets="All" GeneratePathProperty="true"/>
```

Comment:
VB?
```suggestion
    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeStyle" Version="$(MicrosoftCodeAnalysisCSharpCodeStylePackageVersion)" ExcludeAssets="All" GeneratePathProperty="true"/>
    <PackageReference Include="Microsoft.CodeAnalysis.ViusalBasic.CodeStyle" Version="$(MicrosoftCodeAnalysisViusalBasicCodeStylePackageVersion)" ExcludeAssets="All" GeneratePathProperty="true"/>
```

Reply:
What is the name of the codestyle package for VB? I couldn't find one

Reply:
https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-tools&package=Microsoft.CodeAnalysis.VisualBasic.CodeStyle&protocolType=NuGet&version=3.8.0-3.20418.5